.github/workflows/release-backend.yml

```diff
@@ -0,0 +1,13 @@
+on:
+  push:
+    tags:
+      - ccdscan-fungus/*
```

Comment:
https://concordium.slack.com/archives/C03DRCC9GR1/p1730893953908189

Seems to be consensus

Reply:
I thought people were just joking and coming up with funny names. it could be more confusing why we call it `fungus/fungy`, so my suggestion is to be super boring and call it `backend/` or if we want to avoid clashes with previous tags `backend-rust`.

Reply:
sure, I have also adjusted the repository name on docker hub

Reply:
also I called it ccdscan-backend, since that correlates quite nicely with the ccdscan-frontend, and the current backends service name is ccdscan. So we can at a later point remove that to remove confusion